### PR TITLE
setup cypress configuration

### PR DESCRIPTION
**Pull request by MariaJose319**

### CODE REVIEW

File: cypress/support/commands.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+// -- This is a child command --
+// Cypress.Commands.add('drag', { prevSubject: 'element'}, (subject, options) => { ... })
+//
+//
+// -- This is a dual command --
+// Cypress.Commands.add('dismiss', { prevSubject: 'optional'}, (subject, options) => { ... })
+//
+//
+// -- This will overwrite an existing command --
+// Cypress.Commands.overwrite('visit', (originalFn, url, options) => { ... })
```
Comment (AlejaGonzalezV) [inline cypress/support/commands.js]: Please add an empty line at the end of the file (EOF)

File: cypress/support/e2e.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+//
+// You can read more here:
+// https://on.cypress.io/configuration
+// ***********************************************************
+
+// Import commands.js using ES2015 syntax:
+import './commands'
+
+// Alternatively you can use CommonJS syntax:
+// require('./commands')
```
Comment (AlejaGonzalezV) [inline cypress/support/e2e.js]: Please add an empty line at the end of the file (EOF)

**Review by AlejaGonzalezV — COMMENTED** [review]
Why do you have a file named downloads.html? 🤔

**Comment by MariaJose319** [comment]
> Why do you have a file named downloads.html? 🤔

hehe...I don't have any idea. Should I delete it?

**Comment by AlejaGonzalezV** [comment]
> > Why do you have a file named downloads.html? 🤔
> 
> hehe...I don't have any idea. Should I delete it?

Yess, it should not be there 😄

**Review by AlejaGonzalezV — APPROVED** [review]

**Pull request merged**